### ISSUE TITLE
Move <abbr> from unapi implementation in catalog/_index_default out of ul

### ISSUE THREAD

**Issue by axamei** [pulibrary/orangelight#1063]
Violation: `ul` and `ol` must only directly contain `<li> <script> or <template>` elements

Can we move this [abbr tag](https://github.com/pulibrary/orangelight/blob/1e937344e629592d7122dc2933cb70d4bace1d94/app/views/catalog/_index_default.html.erb#L4) elsewhere in the record-wrapper? It cannot be a direct child of `ul`
